context/app/static/js/shared-styles/buttons/OutlinedLinkButton/OutlinedLinkButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { PropsWithChildren } from 'react';
+import Button from '@mui/material/Button';
+import LinkIcon from '@mui/icons-material/Link';
+import SvgIcon from '@mui/material/SvgIcon';
+import Typography from '@mui/material/Typography';
+
+interface OutlinedLinkButtonProps extends PropsWithChildren {
+  link: string;
+}
+
+function OutlinedLinkButton({ link, children }: OutlinedLinkButtonProps) {
+  return (
+    <Button
+      variant="outlined"
+      color="info"
+      sx={(theme) => ({
+        borderColor: theme.palette.grey[300],
+        borderRadius: theme.spacing(0.5),
+      })}
+      href={link}
+    >
+      <Typography variant="inherit" marginRight=".5rem">
+        {children}
+      </Typography>
+      <SvgIcon component={LinkIcon} color="info" sx={{ fontSize: '1rem' }} />
```

Comment:
I think we can pass this in as an `endIcon` on the Button itself: https://mui.com/material-ui/api/button/#button-prop-endIcon

We may also be able to directly use the LinkIcon component instead of relying on `<SvgIcon component={...}/>` since it's a consistent icon, but that's not a huge deal.

Reply:
Ah nice!